worker-queue.yml

```diff
@@ -18,7 +18,22 @@ Parameters:
   WorkersSecurityGroup:
     Type: AWS::EC2::SecurityGroup::Id
 
+  QueueAddress:
```

Comment:
I'd alphabetize the list of parameters (putting QueueAddress between KeyPair and Subnet).